sys-utils/nsenter.1.adoc

```diff
@@ -137,6 +137,9 @@ Do not fork before exec'ing the specified program. By default, when entering a P
 *-Z*, *--follow-context*::
 Set the SELinux security context used for executing a new process according to already running process specified by *--target* PID. (The util-linux has to be compiled with SELinux support otherwise the option is unavailable.)
 
+*-c*, *--join-cgroup*::
+Allow the process initiated by *nsenter* to join the cgroup of the target process.
```

Comment:
What does "Allow" mean here?

The initiated process doesn't have a choice, or? So it would be:

"Add the initiated process to the cgroup of the target process".

Reply:
Thank you for your feedback. "Add the initiated process to" is more clearly states the action being performed.

I will amend the statement accordingly.